src/main/java/org/checkstyle/autofix/CheckstyleAutoFix.java

```diff
@@ -42,6 +42,12 @@ public class CheckstyleAutoFix extends Recipe {
             example = "config/checkstyle.xml")
     private String configurationPath;
 
+    @Option(displayName = "Checkstyle properties file path",
+            description = "Path to the file containing the Checkstyle Properties.",
+            example = "config/checkstyle.properties",
+            required = false)
+    private String propertiesPath;
```

Comment:
I assume, this property is not mandatory. Could you check how OpenRewrite suggests to implement optional properties? Here we'll get null if it is not set, but I'm curious if we can mark it with some optional annotation or make it return `Optional<String>`.